docs/examples/here-data-sdk-aaa-example/README.md

```diff
@@ -0,0 +1,42 @@
+# Simple Authorization WEB App 
+
+This is a simple Web App with configured compiling TS to JS and bundling with webpack.
+The app shows how to use APIs from `@here/olp-sdk-dataservice-api`; 
+
+All function from AuthorizationAPI can be used in the same way as used in this example.
+
+## Setup
+
+1) Download the current folder to your PC.
+2) Add to your enviropment app key and secret.
+```
+export OLP_APP_KEY=<your app key>
+export OLP_APP_SECRET=<your app secret>
+```
+3) Install dependencies.
+
+```
+npm install
+```
+
+## Build and run
+
+1) Compile TS to JS.
+
+```
+npm run compile
+```
+
+2) Bundle with webpack.
+
+```
+npm run build
+```
+
+3) Start web server.
+
+```
+npm start
+```
+
+4) Open in your favorite browser http://localhost:8080
```

Comment:
```suggestion
## Build and run

1. Compile TS files into JS files.

    ```
    npm run compile
    ```

2. Bundle with webpack.

    ```
    npm run build
    ```

3. Start the web server.

    ```
    npm start
    ```

4. Open the web server in your favorite browser: http://localhost:8080
```

Reply:
Open the web server in - not correct, ignored that.

Reply:
Please change the formatting.